### DISCUSSION TITLE
AC-41 Purifier

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#37] [Card Balance]
EQUIPMENT FIREARM
S +3

**Comment by Sol-au-fein** [comment]
The Purifier is combination of light machine gun and an incinerator. It is very nasty weapon, let down only by its relative short range. The Purifier is a heavy weapon and its integral incinerator is a flamethrower.

Suggestion:
S +2, opponents A -2

**Reply by Sol-au-fein** [reply]
This might need some work. It is not very attractive.

**Comment by 21-ric** [comment]
Because it's a nasty weapon. 🤡

AC-41 'Purifier' (BROTHERHOOD) [U]
EQUIPMENT, FIREARM
[S] +3. Opponent warrior may not gain bonus from commander during shoot combat.

**Reply by Sol-au-fein** [reply]
Too marginal :/ you would be lucky to use it once in lifetime. Good idea though, i think some other card could have this effect.

**Reply by 21-ric** [reply]
You are right. It's not easy to find the best suggestion for this card. If we are lazy we give this card +5 to [S] and SLAYER. But we want more. This card will be our Mona Lisa. 🤡

**Comment by Sol-au-fein** [comment]
So @framerate will most likely add BURN to Purifier. How to differenciate it from Plasma Carbine? In another words why would brotherhood warriors want to pick Purifier instead of Plasma Carbine?

**Reply by 21-ric** [reply]
If we give +2 to [S] and burn 1 for "L&A Plasma Carbine".
We could give +3 to [S] and burn 1 for "AC-41 'Purifier'". 

Let's wait feedback about burn. We could need a new card for this ability.

**Reply by Kite80** [reply]
I agree with giving AC-41 BURN skill

**Reply by Sol-au-fein** [reply]
I like when Carbine is +3. What if Purifier was +5?

**Comment by Sol-au-fein** [comment]
So everyone agree with S +5 + BURN?
